libgadget/tests/test_forcetree.c

```diff
@@ -226,7 +226,9 @@ static void do_tree_test(const int numpart, ForceTree tb, DomainDecomp * ddecomp
     /*Time creating the nodes*/
     double start, end;
     start = MPI_Wtime();
-    int nodes = force_tree_create_nodes(tb, numpart, ALLMASK, ddecomp, 0);
+    ActiveParticles Act = {0};
+    Act.NumActiveParticle = numpart;
```

Comment:
Is it necesary to set numactiveparticle on Act = {0}? I noticed above the NumActiveParticle line is omitted.

Reply:
That is a good point - it should not be necessary at the moment but I have been trying to do it in case that changes. This should be a proper function so it is consistent everywhere, and now it is.